test/apis/realtime/multi-container/app/main.py

```diff
@@ -0,0 +1,40 @@
+import base64
+import requests
+
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+
+class Request(BaseModel):
+    image_url: str
+
+
+app = FastAPI()
+
+
+@app.get("/healthz")
+def healthz():
+    return "ok"
+
+
+@app.post("/")
+def text_generator(request: Request):
+    server_url = f"http://localhost:8501/v1/models/resnet50:predict"
+
+    # download labels
+    labels = requests.get(
+        "https://storage.googleapis.com/download.tensorflow.org/data/ImageNetLabels.txt"
+    ).text.split("\n")[1:]
```

Comment:
Should these be moved outside of the `text_generator()` method?

Reply:
Yep, they can. Same can be said about `server_url`.